src/app/components/course-payment/enroll-student/enroll-student.component.html

```diff
@@ -17,10 +17,16 @@
         <div class="row d-flex justify-content-center">
           <mat-form-field class="col-md-4">
             <input matInput placeholder="First name" formControlName="firstName" required>
+            <mat-error *ngIf="firstFormGroup.controls['firstName'].hasError('required') && firstFormGroup.controls['firstName'].touched">
+              Please fill this field
```

Comment:
can we have some user friendly commands like "please fill firstname" or "First name is required"